### DISCUSSION TITLE
How can i remove the lines

### DISCUSSION THREAD

**Discussion by cuchoco** [mmistakes/minimal-mistakes#3526] [Q&A]
![image](https://user-images.githubusercontent.com/37405384/158532204-23869f9c-92b0-45eb-8ce4-9dd56cd94b38.png)

when i commit new post
sometimes unidentified lines are formed along the texts like picture
i don't know whats the matter ..

**Comment by iBug** [comment]
The theme does not come with such things. Unless you provide your website and the repository, there's nothing we can diagnose.

**Reply by cuchoco** [reply]
my website: cuchoco.github.io
repo : https://github.com/cuchoco/cuchoco.github.io

my theme is 'dirt' and i revised some configures according to other blog posts 
To be honest, I don't know much about jekyll.

**Reply by iBug** [reply]
This is where you're doing wrong:

<https://github.com/cuchoco/cuchoco.github.io/commit/9cc29931bb4afacb62f52ae8ee366f8c7e5a6f3e>

CSS is complex so you're on your own now.
